core/domain/suggestion_services.py

```diff
@@ -85,11 +86,9 @@ def create_suggestion(
     """
     if description is None:
         description = DEFAULT_SUGGESTION_THREAD_SUBJECT
-    thread_id = feedback_services.create_thread(
-        target_type, target_id, author_id, description,
-        DEFAULT_SUGGESTION_THREAD_INITIAL_MESSAGE, has_suggestion=True)
 
     status = suggestion_models.STATUS_IN_REVIEW
+    should_send_emails = False
```

Comment:
This appears to be a functional change--we no longer want to send emails for the other suggestion types. Can you provide some context as to why?